test_util/aws.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import logging
+from collections import namedtuple
+
+import boto3
+
+LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
+logging.basicConfig(format=LOGGING_FORMAT, level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+Host = namedtuple('Host', ['private_ip', 'public_ip'])
```

Comment:
:+1: Eventually we should sink this into the general test_util / runner libraries probably (Azure, GCE have the same things)